demos/nytimes.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+#
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script uses the [OpenAPI Directory](https://github.com/APIs-guru/openapi-directory)
+# to build a collection of APIs and then performs some analysis on the uploaded API
+# descriptions. It assumes that this repo has been cloned to the user's Desktop,
+# so that it can be found at `~/Desktop/openapi-directory`.
+
+# It also assumes an environment configured to call a Registry API implementation.
+# This includes the registry-server running with a local SQLite database,
+# which can be started by running `registry-server -c config/sqlite.yaml`
+# from the root of the registry repo. To configure clients to call this
+# server, run `source auth/LOCAL.sh` in the shell before running the following
+# commands.
+
+# First, delete the "nytimes" project to get a fresh start.
+apg registry delete-project --name projects/nytimes
+
+# Get the commit hash of the checked-out OpenAPI directory
+export COMMIT=`(cd ~/Desktop/openapi-directory; git rev-parse HEAD)`
+
+# Upload all of the APIs in the OpenAPI directory.
+registry upload bulk openapi \
+	--project_id nytimes ~/Desktop/openapi-directory/APIs/nytimes.com \
+	--base_uri https://github.com/APIs-guru/openapi-directory/blob/$COMMIT/APIs/nytimes.com 
+
+# The nytimes project was automatically created. Here we'll use an
+# update-project call to set a few properties of the project.
+apg registry update-project \
+	--project.name "projects/nytimes" \
+	--project.display_name "New York Times APIs" \
+	--project.description "APIs collected from the APIs.guru OpenAPI Directory"
+
+# Now compute summary details of all of the APIs in the project. 
+# This will log errors if any of the API specs can't be parsed,
+# but for every spec that is parsed, this will set the display name
+# and description of the corresponding API from the values in the specs.
+registry compute details projects/nytimes/apis/-
+
+# Verify this for one of the APIs.
+registry get projects/nytimes/apis/archive
```

Comment:
`locations/global` is missing in resource names in this entire file.

Reply:
Thanks for catching this - I'm going to remove this and the other organization-specific demo scripts from this PR since they weren't already in the repo

Reply:
Removed